spec/unit/plugins/dmi_spec.rb

```diff
@@ -91,6 +91,10 @@
 	Contained Elements: 0
 EOS
 
+def convert_window_output(stdout)
```

Comment:
This seems useful enough to put into a `spec_helper` of some kind - I would normally only advocate this if something else already needed this functionality, but I think putting it in a helper will encourage future spec writers to leverage it if they see it being used here.